### ISSUE TITLE
Some vanilla commands cause a panic for command macro.

### ISSUE THREAD

**Issue by Miro-Andrin** [feather-rs/lieutenant#14]
In my quest to implement not implemented msg for all vanilla commands, i  have stumbled across a couple of cases were the lieutenants command macro panics. 

---------------------------------------
```
#[command(usage="execute if score <target> <targetObjective> = <source> <sourceObjective>")]
pub fn execute_if_score_target_targetObjective_eq_source_sourceObjective(
    ctx: &mut CommandCtx,
    target:SingleScoreHolder,
    targetObjective:Objective,
    source:SingleScoreHolder,
    sourceObjective:Objective
) -> anyhow::Result<()> {
    if let Some(mut sender_message_receiver) = ctx.world.try_get_mut::<MessageReceiver>(ctx.sender)
    {
        let return_text = Text::from("This command \"execute if score <target> <targetObjective> = <source> <sourceObjective>\" is not implemented in this version of feather.").gray().italic();
        sender_message_receiver.send(return_text);
    }
    Ok(Some("".to_string()))
}
```

------------------------------------------------------
For this one (below) the command compiles if the greater then symbol is removed/changed. 

```
#[command(usage="execute if score <target> <targetObjective> > <source> <sourceObjective>")]
pub fn execute_if_score_target_targetObjective_gt_source_sourceObjective(
    ctx: &mut CommandCtx,
    target:SingleScoreHolder,
    targetObjective:Objective,
    source:SingleScoreHolder,
    sourceObjective:Objective
) -> anyhow::Result<()> {
    if let Some(mut sender_message_receiver) = ctx.world.try_get_mut::<MessageReceiver>(ctx.sender)
    {
        let return_text = Text::from("This command \"execute if score <target> <targetObjective> > <source> <sourceObjective>\" is not implemented in this version of feather.").gray().italic();
        sender_message_receiver.send(return_text);
    }
    Ok(Some("".to_string()))
}
```
--------------------------------
For this one (below) it does not panic if the less then symbol in the command macro is replaced/removed. 
```
#[command(usage="execute unless score <target> <targetObjective> < <source> <sourceObjective>")]
pub fn execute_unless_score_target_targetObjective_lt_source_sourceObjective(
    ctx: &mut CommandCtx,
    target:SingleScoreHolder,
    targetObjective:Objective,
    source:SingleScoreHolder,
    sourceObjective:Objective
) -> anyhow::Result<()> {
    if let Some(mut sender_message_receiver) = ctx.world.try_get_mut::<MessageReceiver>(ctx.sender)
    {
        let return_text = Text::from("This command \"execute unless score <target> <targetObjective> < <source> <sourceObjective>\" is not implemented in this version of feather.").gray().italic();
        sender_message_receiver.send(return_text);
    }
    Ok(Some("".to_string()))
}
```
---------------------------------------
For this one (below) the command does not panic when = is substituted with 'eq'. 
making the macro call into:
`#[command(usage="execute unless score <target> <targetObjective> eq <source> <sourceObjective>")]`

```
#[command(usage="execute unless score <target> <targetObjective> = <source> <sourceObjective>")]
pub fn execute_unless_score_target_targetObjective_eq_source_sourceObjective(
    ctx: &mut CommandCtx,
    target:SingleScoreHolder,
    targetObjective:Objective,
    source:SingleScoreHolder,
    sourceObjective:Objective
) -> anyhow::Result<()> {
    if let Some(mut sender_message_receiver) = ctx.world.try_get_mut::<MessageReceiver>(ctx.sender)
    {
        let return_text = Text::from("This command \"execute unless score <target> <targetObjective> = <source> <sourceObjective>\" is not implemented in this version of feather.").gray().italic();
        sender_message_receiver.send(return_text);
    }
    Ok(Some("".to_string()))
}
```
----------------------------------------------
For this one (below) the command does not panic when > is substituted with 'lt'. 

```
#[command(usage="execute unless score <target> <targetObjective> > <source> <sourceObjective>")]
pub fn execute_unless_score_target_targetObjective_gt_source_sourceObjective(
    ctx: &mut CommandCtx,
    target:SingleScoreHolder,
    targetObjective:Objective,
    source:SingleScoreHolder,
    sourceObjective:Objective
) -> anyhow::Result<()> {
    if let Some(mut sender_message_receiver) = ctx.world.try_get_mut::<MessageReceiver>(ctx.sender)
    {
        let return_text = Text::from("This command \"execute unless score <target> <targetObjective> > <source> <sourceObjective>\" is not implemented in this version of feather.").gray().italic();
        sender_message_receiver.send(return_text);
    }
    Ok(Some("".to_string()))
}
```
----------------------------------------------
For this one (below) the command does not panic when * is substituted with 'star'. 
`#[command(usage="recipe give <targets> star")]`

```
#[command(usage="recipe give <targets> *")]
pub fn recipe_give_targets_star(
    ctx: &mut CommandCtx,
    targets:MultiplePlayers
) -> anyhow::Result<()> {
    if let Some(mut sender_message_receiver) = ctx.world.try_get_mut::<MessageReceiver>(ctx.sender)
    {
        let return_text = Text::from("This command \"recipe give <targets> *\" is not implemented in this version of feather.").gray().italic();
        sender_message_receiver.send(return_text);
    }
    Ok(Some("".to_string()))
}
```
-------------------------------
For this one (below) the command does not panic when * is substituted with 'star'. 

```
#[command(usage="recipe take <targets> *")]
pub fn recipe_take_targets_star(
    ctx: &mut CommandCtx,
    targets:MultiplePlayers
) -> anyhow::Result<()> {
    if let Some(mut sender_message_receiver) = ctx.world.try_get_mut::<MessageReceiver>(ctx.sender)
    {
        let return_text = Text::from("This command \"recipe take <targets> *\" is not implemented in this version of feather.").gray().italic();
        sender_message_receiver.send(return_text);
    }
    Ok(Some("".to_string()))
}
```
--------------------------------------
For this one (below) the command does not panic when * is substituted with 'star'. 

```
#[command(usage="stopsound <targets> * <sound>")]
pub fn stopsound_targets_star_sound(
    ctx: &mut CommandCtx,
    targets:MultiplePlayers,
    sound:ResourceLocation
) -> anyhow::Result<()> {
    if let Some(mut sender_message_receiver) = ctx.world.try_get_mut::<MessageReceiver>(ctx.sender)
    {
        let return_text = Text::from("This command \"stopsound <targets> * <sound>\" is not implemented in this version of feather.").gray().italic();
        sender_message_receiver.send(return_text);
    }
    Ok(Some("".to_string()))
}
```

----------------------------------

For the one below it does not panic when the first literal 'w' is changed to 'wa', but does panic if it is changes to one of 
'a', 'b', 'z'. My guess is that the macro or the code handeling literals cant handle literals containing just one char. 

-----------------------------------

```
#[command(usage="w <targets> <message>")]
pub fn w_targets_message(
    ctx: &mut CommandCtx,
    targets:MultiplePlayers,
    message:Message
) -> anyhow::Result<()> {
    if let Some(mut sender_message_receiver) = ctx.world.try_get_mut::<MessageReceiver>(ctx.sender)
    {
        let return_text = Text::from("This command \"w <targets> <message>\" is not implemented in this version of feather.").gray().italic();
        sender_message_receiver.send(return_text);
    }
    Ok(Some("".to_string()))
}
```

-------------------------------------------------

All the error msg's are "attempt to subtract with overflow."

**Comment by caelunshun** [comment]
Thanks, I'll fix these errors today/tomorrow. Sorry about the delay!

**Comment by caelunshun** [comment]
Thanks, I'll fix these errors today/tomorrow. Sorry about the delay!

**Comment by caelunshun** [comment]
I seem to have fixed the problem in ef7d260. Thanks for the comprehensive report, and let me know if you find any more issues!
